### DISCUSSION TITLE
How to insert a text between two form fields?

### DISCUSSION THREAD

**Discussion by njxqlus** [filamentphp/filament#7659] [Help]
I have a Filament Resource with form:
```php
public static function form(Form $form): Form
    {
        return $form
            ->columns(1)
            ->schema([
                Forms\Components\Section::make()->schema([
                    Forms\Components\TextInput::make('name'),
                    Forms\Components\MarkdownEditor::make('description'),
                ])
            ]);
    }
```

Can't find in documentation how to add a simple text, image or custom html between fields `name` and `description`?

The only way I found is to add a `\Filament\Forms\Components\View::make('components.my-custom-text')` but it looks like a bit overwhelming.

Am I miss something or this the the concept?

**Comment by danharrin** [comment]
Which bit is overwhelming? You create a Blade file, and put whatever HTML you want in it. It's the simplest way to do this and the most flexible.

**Reply by njxqlus** [reply]
Overwhelming is create a file for one `<div>` or `<hr>`. I think maybe there is maybe a way like `Html::make('<p>Hello</p>')`.

Also it's annoying to create name for this files. Like `text-above-name-field`

**Reply by mikestmike** [reply]
https://filamentphp.com/docs/3.x/forms/layout/placeholder

use Filament\Forms\Components\Placeholder;
use Illuminate\Support\HtmlString;
 
Placeholder::make('documentation')
    ->content(new HtmlString('<a href="https://filamentphp.com/docs">filamentphp.com</a>'))

**Comment by gilr** [comment]
I agree, this is a bit overwhelming for a small text... But I needed it too, and the Placeholder exists 😃:
(updated with hiddenLabe()l following the last answer from danharrin)

`        Placeholder::make('My Label')->content("blah blah"),
`

or with only one line without label, and included in its own section:

`       Section::make()->schema([ Placeholder::make('No Label')->hiddenLabel()->content("blah blah") ]),
`

[https://filamentphp.com/docs/3.x/forms/layout/placeholder](url)

**Reply by njxqlus** [reply]
It's look like hack because the documentation says that **Important: All form fields require a unique name. That also applies to Placeholders!**

**Reply by gilr** [reply]
Oh yes, right! Or a documentation issue, no sé... 
As the Placeholder is technically in the "layout" documentation chapter, and the section in the same chapter does not require a name. 
The fieldset also can work without label (even not documented), it looks like a section, not reallly useful.

![image](https://github.com/filamentphp/filament/assets/2454046/5b30ac86-5da1-49fd-a9c4-3f269a137155)

**Reply by danharrin** [reply]
Please include a label, as the docs say. You can hide it again with `hiddenLabel()`. But it should be defined, or it will cause issues.

**Reply by gilr** [reply]
Fine thanks!, I updated my first answer to be complete and correct.
